projects/valtimo/dossier/src/lib/dossier-detail/tab/summary/summary.component.ts

```diff
@@ -151,4 +156,23 @@ export class DossierDetailTabSummaryComponent implements OnInit, OnDestroy {
   public rowTaskClick(task: any) {
     this.taskDetail.openTaskDetails(task);
   }
+
+  private unescapeComponentDefaultValue = (
+    component: ExtendedComponentSchema
+  ): ExtendedComponentSchema => {
+    if (component.defaultValue) {
+      return {...component, defaultValue: this.unescape(component.defaultValue)};
+    }
+
+    return component;
+  };
+
+  private unescape(htmlStr: string): string {
```

Comment:
This doesn't handle all escaped characters. See: https://mateam.net/html-escape-characters/